### PR TITLE
update to 0.12.2

### PR DESCRIPTION
**Pull request by philip-khor**
Per conda-forge/r-skynet-feedstock#5

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -48,7 +47,7 @@ test:
 
 about:
   home: http://corybrunson.github.io/ggalluvial/
```
Comment (Anthchirp) [inline recipe/meta.yaml]:
```suggestion
  home: https://cloud.r-project.org/package=ggalluvial
  doc_url: https://corybrunson.github.io/ggalluvial/
```

**Review by Anthchirp — COMMENTED** [review]
Would also suggest linking the DOIs in the summary using
```markdown
  summary: Alluvial diagrams use x-splines, sometimes augmented with stacked histograms, to visualize
    multi-dimensional or repeated-measures data with categorical or ordinal variables.
    They can be viewed as simplified and standardized Sankey diagrams; see Riehmann,
    Hanfler, and Froehlich (2005) [doi:10.1109/INFVIS.2005.1532152](https://doi.org/10.1109/INFVIS.2005.1532152)
    and Rosvall and Bergstrom (2010) [doi:10.1371/journal.pone.0008694](https://doi.org/10.1371/journal.pone.0008694).
    This package provides ggplot2 layers to produce alluvial diagrams from tidy data.
```

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Pull request merged**